ui/raidboss/data/05-shb/raid/e12s.ts

```diff
@@ -1076,12 +1076,7 @@ const triggerSet: TriggerSet<Data> = {
           ko: '멀리 유도하기',
         },
         partyUnder: {
-          en: 'Get Under',
-          de: 'Unter ihn',
-          fr: 'En dessous',
-          ja: 'ボスと貼り付く',
-          cn: '去脚下',
-          ko: '보스 안쪽으로',
+          en: 'Party Under',
```

Comment:
Maybe these calls should be like the Apo/Peri calls in p12s, where tanks get "Tank Out (Party In)" and party gets "Party In (Tanks Out)" and BLU could just get either one of those, honestly.

Reply:
They probably should be, but I was going for minimum effort changes just to have something, and also make it less confusing for the tank BLUs.

Reply:
Would you mind just replacing with the calls from Apo/Peri? If this were an older older fight I'd say whatever, but this is something that is going to be done on blu very shortly.